pulp_ansible/app/serializers.py

```diff
@@ -102,6 +102,15 @@ class AnsibleRepositorySyncURLSerializer(RepositorySyncURLSerializer):
     optimize = serializers.BooleanField(
         help_text=_("Whether to optimize sync or not."), default=True
     )
+    excludes_file = serializers.CharField(
```

Comment:
do we still need this on the serializer now that `excludes` are being passed to sync function as a parameter?

Reply:
Yes, either way it needs to be there.

Reply:
I believe we should have this data stored on the Remote. The high-level design is that all meaningful details for sync live on the Remote. We started introducing these sync-call options because remotes could be reused between multiple repositories and it became clear passing them at runtime was a better option, mirror. Or for things that are truly the functionality of the sync, e.g. `optimize`.